content/gotraining.slide

```diff
@@ -425,6 +429,53 @@ As part of all labs
 - Create a PR following the [[https://github.com/anz-bank/go-training#pre-pr-checklist][PR checklist]]
 
 
+* PR from own fork - initial setup
+
+Issue PRs from `feature` branch of your own fork to `master` on upstream:
+
+- Login to github.com
+- Click `Fork` in the top right corner of [[https://github.com/anz-bank/go-training-slides]]
```

Comment:
Is this meant to be `go-training` or `go-slides`? `go-training-slides` as you have it works, but it redirects to `go-slides`, so when you fork it, your repo is called `go-slides`. This looks like it should break the origin URL below.

I feel like this should be `go-training` since you reference the pre-PR checklist from that repo just above, but not sure. I'm going to assume `go-training` for the purposes of my comments below (or above depending on the order github shows these).

Reply:
Mistake.
Done.